etc/kayobe/ansible/templates/os_capacity-clouds.yml.j2

```diff
@@ -1,10 +1,13 @@
 clouds:
   openstack:
     auth:
-      auth_url: "{{ secrets_os_exporter_auth_url }}"
-      application_credential_id: "{{ secrets_os_exporter_credential_id }}"
-      application_credential_secret: "{{ secrets_os_exporter_credential_secret }}"
+      auth_url: "{{ secrets_os_capacity_auth_url }}"
+      application_credential_id: "{{ secrets_os_capacity_credential_id }}"
+      application_credential_secret: "{{ secrets_os_capacity_credential_secret }}"
     region_name: "RegionOne"
     interface: "internal"
     identity_api_version: 3
     auth_type: "v3applicationcredential"
+{% if kolla_enable_tls_internal | bool %}
+    verify: False
```

Comment:
Is there no way we can copy or bind mount the CA certificate file into the container?

Reply:
Moving TLS verification to #896